agent-jenkins-tools/Dockerfile

```diff
@@ -3,42 +3,29 @@ FROM quay.io/feedhenry/jenkins-agent-java:latest
 USER root
 
 # Jenkins Job Builder
-
 ENV BRANCH master
 
 WORKDIR /
-RUN git clone -b allow_self_signed_certs https://github.com/mikenairn/python-jenkins.git && \
```

Comment:
Is there a reason why we no longer clone this repo?

Reply:
Yeah, we don't need it any longer. :) We used to need that custom version of python-jenkins to be able to deal with self-signed certs, but the main version has since gotten a better way to do that, by looking for an environment variable. See here: https://github.com/openstack/python-jenkins/commit/5d9d6dd4a00ea46cd1322dc84accb07b2f724d95

Reply:
Good to know 👍